applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -394,6 +395,95 @@ void BaseSolidElement::CalculateRightHandSide(
     CalculateAll( temp, rRightHandSideVector, rCurrentProcessInfo, CalculateStiffnessMatrixFlag, CalculateResidualVectorFlag );
 }
 
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void BaseSolidElement::AddExplicitContribution(
+    const VectorType& rRHSVector, 
+    const Variable<VectorType>& rRHSVariable,
+    Variable<double>& rDestinationVariable,
+    const ProcessInfo& rCurrentProcessInfo
+    ) 
+{
+    KRATOS_TRY;
+
+    auto& r_geom = this->GetGeometry();
+    const SizeType dimension = r_geom.WorkingSpaceDimension();
+    const SizeType number_of_nodes = r_geom.size();
+    const SizeType element_size = dimension * number_of_nodes;
+
+    // Compiting the nodal mass
+    if (rDestinationVariable == NODAL_MASS ) {
+        Matrix element_mass_matrix = ZeroMatrix(element_size, element_size);
+        this->CalculateLumpedMassMatrix(element_mass_matrix);
```

Comment:
do you always want to compute the Lumped mass-matrix in this Function? Consistent one is not an option here?

Reply:
It is explicit, to being full parallel the lumped matrix is always assumed